xmtp/Cargo.toml

```diff
@@ -36,7 +36,7 @@ ethers-core = "2.0.4"
 prost = { version = "0.11", features = ["prost-derive"] }
 futures = "0.3.28"
 base64 = "0.21.1"
-tokio = "1.28.1"
```

Comment:
These changes are needed to make the tests pass when running `cargo test` from inside the xmtp crate directly